dll/win32/msi/custom.c

```diff
@@ -867,6 +867,17 @@ static UINT HANDLE_CustomType1( MSIPACKAGE *package, const WCHAR *source, const
     if (!(binary = get_temp_binary(package, source)))
         return ERROR_FUNCTION_FAILED;
 
+#if defined(__REACTOS__) && defined(_M_AMD64)
+    {
+        DWORD arch;
+        get_binary_type(binary->tmpfile, &arch);
+        if (arch == SCS_32BIT_BINARY) {
+            ERR("%s is a 32 bit custom action. Returning as ERROR_SUCCESS\n", debugstr_w( source ));
```

Comment:
```suggestion
            ERR("%s is a 32 bit custom action. Returning as ERROR_SUCCESS\n", debugstr_w(source));
```
see coding guidelines.